mods/ts/rules/gdi-support.yaml

```diff
@@ -128,6 +134,8 @@ GACTWR:
 			tower.sam: tower.sam
 	ProvidesPrerequisite@buildingname:
 	SelectionDecorations:
+	Replacement:
+		Replaces: gawall
```

Comment:
`gawall` seamlessly connects to `nawall`, so I don't see any real reason to restrict tower or gate types to specific wall types.  It would be better to have a single `Wall` replacement type shared by both.

Reply:
I tried both ending up with this seperation only becouse it seemed "better" wirh the gdi turret.
Will changr it to wall as recommendet

Reply:
>`gawall` seamlessly connects to `nawall`

Well, that's not how TS worked. This is the TS-accurate setup.

Reply:
I'll have a look at vanilla-SUN.
Think it's up for discussion. Changing the replaceable would be a simple yaml file fix, anyway.

Should we change LineBuild to distinct `nawall` and `gawall`?

Reply:
I'm not fussed either way.  It seems silly to not allow cross-faction replacement, but as it will be a rare edge case in practice I don't see any reason to cause drama by insisting on it.